Doc/library/annotationlib.rst

```diff
@@ -603,3 +607,22 @@ Below are a few examples of the behavior with unsupported expressions:
    >>> def ifexp(x: 1 if y else 0): ...
    >>> get_annotations(ifexp, format=Format.STRING)
    {'x': '1'}
+
+.. _annotationlib-security:
+
+Security
+--------
+
+Much of the functionality in this module involves executing arbitrary code. For example,
```

Comment:
```suggestion
Much of the functionality in this module involves executing code in annotations, which can then do arbitrary things
```

I think currently it reads a little scarier than it should. I am much less scared seeing get_annotations than seeing eval, and I think it's worth being a little bit more specific about what is "arbitrary".

(I don't know that my specific suggestion here is that good though)